programs/vote/src/vote_state.rs

```diff
@@ -334,6 +346,21 @@ impl VoteState {
             }
         }
     }
+
+    pub fn process_timestamp(
+        &mut self,
+        slot: Slot,
+        timestamp: UnixTimestamp,
+    ) -> Result<(), VoteError> {
+        if (slot < self.last_timestamp.0 || timestamp < self.last_timestamp.1)
```

Comment:
I'm non a fan of using tuples for public fields, I'd much prefer a struct because:

1. I keep having to refer back to the struct because I forget what .0 and .1 are
2. I get confused when I see things like `slot == self.last_timestamp.0`.  My brain likes to ignore the insignificant looking `.0` suffix and then wonders why we're comparing a slot to a timestamp.